admin-api-server/src/nft/nft.service.ts

```diff
@@ -188,23 +203,42 @@ ORDER BY id
       // we'll simply store the uri as a pointer to the image in our own db
       const contentUri = await this.s3Service.uploadFile(picture, fileName);
 
+      const updatableNft = {
+        attribute: this.CONTENT_KEYWORD,
+        value: JSON.stringify(contentUri)
+      }
+
       return await this.apply(
         user,
         nftId,
-        this.CONTENT_KEYWORD,
-        JSON.stringify(contentUri),
+        [updatableNft],
         false,
       );
     } finally {
       this.nftLock.release(nftId);
     }
   }
 
+  transformFormDataToUpdatableNft(attrArray: string[], valueArray: any | any[]) {
```

Comment:
In this encoding, how to unset an attribute? In the previous approach (ie before this PR), it was possible to pass an attribute, without setting the value. like this:

```
  http PATCH /nft/1 <<< '
    {
      "attribute": "someAttribute"
    }
```